src/packages/sdk/src/engine-commands/getConfig.ts

```diff
@@ -0,0 +1,156 @@
+import Debug from '@prisma/debug'
+import { NApiEngineTypes } from '@prisma/engine-core'
+import { EngineTypes } from '@prisma/fetch-engine'
+import { DataSource, GeneratorConfig } from '@prisma/generator-helper'
+import chalk from 'chalk'
+import execa from 'execa'
+import fs from 'fs'
+import tmpWrite from 'temp-write'
+import { promisify } from 'util'
+import { resolveBinary } from '../resolveBinary'
+
+const debug = Debug('prisma:getConfig')
+
+const unlink = promisify(fs.unlink)
+
+const MAX_BUFFER = 1_000_000_000
+
+export interface ConfigMetaFormat {
+  datasources: DataSource[]
+  generators: GeneratorConfig[]
+  warnings: string[]
+}
+
+export type GetConfigOptions = {
+  datamodel: string
+  cwd?: string
+  prismaPath?: string
+  datamodelPath?: string
+  retry?: number
+  ignoreEnvVarErrors?: boolean
+}
+
+export async function getConfig({
+  datamodel,
+  cwd = process.cwd(),
+  prismaPath: queryEnginePath,
+  datamodelPath,
+  ignoreEnvVarErrors,
+}: GetConfigOptions): Promise<ConfigMetaFormat> {
+  const useNapi = process.env.PRISMA_FORCE_NAPI === 'true'
```

Comment:
He we could have getQueryEnginePath

Reply:
What do you mean by this?

Reply:
I assume you mean somthing like is so then this can come later
```ts
funciton getQueryEnginePath(possiblePath: string){
	const engine = getQueryEngine()
	const qePath = resolveBinary(engine, possiblePath)
	return qePath
}
```